### ISSUE TITLE
Accessibility: Add Space key support for Accordions

### ISSUE THREAD

**Issue by jimmieego** [Esri/calcite-web#1062]
According to http://w3c.github.io/aria-practices/#accordion, when focus is on the accordion header, the user should be able to use Enter or Space key to expand or collapse the panel. Right now the Calcite Accordions support the Enter key, but not the Space key.

**Comment by paulcpederson** [comment]
released in 1.2.5
